tests/core/test_merkle_set.py

```diff
@@ -40,14 +40,41 @@ async def test_basics(self, bt):
         assert merkle_set.get_root() == merkle_set_reverse.get_root()
 
 
-prefix = bytes([0] * 30)
+def hashdown(buf: bytes) -> bytes32:
+    return bytes32(sha256(bytes([0] * 30) + buf).digest())
+
+
+@pytest.mark.asyncio
+async def test_merkle_set_invalid_hash_size():
+    merkle_set = MerkleSet()
+
+    # this is too large
+    with pytest.raises(AssertionError):
```

Comment:
shouldn't these be ValueErrors?

Reply:
I think it depends on whether this is considered a programmer error or something that's expected to happen in production. I'm pretty sure it's the former